apps/system/style/net_error.css

```diff
@@ -96,21 +98,29 @@ form[role="dialog"][data-type="action"] > menu > button.icon-settings {
 
 #failed-logo {
   display: inline-block;
-  width: 13.5rem;
   height: 13.5rem;
+  width: 7.9rem;
```

Comment:
It seems a little strange that this doesn't max the pixels of the 1x image, ie 7.4rem. Perhaps you are using that extra space as padding?